### ISSUE TITLE
Deep Learning for Health Informatics

### ISSUE THREAD

**Issue by ghuijo** [yskim0/One-Day-One-Paper#131]
어마어마한 citation 수에 읽어보려다 너무 빽빽해서 미뤄온 논문..
일반 논문 28페이지 같은 느낌의 18페이지 + 리뷰 논문이라 나눠 읽겠습니다....

## Abstract
* "**a comprehensive up-to-date review of research employing deep learning in health informatics**"
* 본 논문에서는 translational bioinformatics, medical imaging, pervasive sensing, medical informatics, public health 분야와 관련된 deep learning 기술을 다룸.

## Introduction
* health informatics 분야에서 가장 많이 쓰이는 deep learning method들 (구글 scholar에서 method 이름 기준으로 수집한 결과)
  <image src="https://user-images.githubusercontent.com/67627471/132128458-929bcdf9-6155-4805-be25-48f2c4caa606.png" width="50%" height="50%" />
* health informatics에 deep learning 기술 적용할 때 문제점
  * training a deep architecture requires an extensive amount of labeled data
  * deep learning requires extensive computational resources
  * deep learning models can be affected by convergence issues as well as overfitting
  * --> 그래서 부가적인 학습 전략이 필요함!

## From Perceptron to Deep Learning
* 말 그대로 perceptron에서 DNN 도입된 과정 설명해줌. (흐름 파악)
* DNN의 챌린지를 아래 표와 함께 설명함.
  ![image](https://user-images.githubusercontent.com/67627471/132129517-4757a363-e1d5-44f1-9817-470302caf7cc.png)
  ![image](https://user-images.githubusercontent.com/67627471/132129535-15fe3be3-2f50-47d4-84d1-614bdfdec547.png)
* 적용되는 health informatics area별로 딥러닝 메쏘드 정리
  <image src="https://user-images.githubusercontent.com/67627471/132129965-87922ecc-80ce-459c-b0e5-b689e174a2d3.png" width="80%" height="80%" />

### Autoencoders and Deep Autoencoders
* Features extracted using data driven learning can generally be more accurate
* 이런 이유에서 Autoencoder가 NN 기반으로 디자인됨.
* class label을 붙이기 보다 input vector를 recreate하기 위해 train됨.
* Deep Autoencoder는 학습 중에 vanishing gradients 문제가 있음.
* 일반적으로 중간에 가중치들을 초기화해서 해결함. 

### Recurrent Neural Network
* NN that contains hidden units capable of analyzing streams of data
* 네트워크에서 시간 t-1일 때 나오는 결과는 시간 t에 영향을 미침.
* RNN은 결과를 낼 때 현재 input과 지난 input(가장 최근)을 모두 input으로 씀.
* RNN과 LSTM을 모든 단계에 걸쳐 같은 가중치들을 공유함.
* bioinformatics 등 자연어 처리 분야에서 뛰어난 성과를 내고 있음.

### RBM-BasedTechnique
* 볼츠만 머신의 변형 (이게 뭔지 모름.. 공부하자!!!)
* modeled by using stochastic units with a specific distribution (가우시안 분포 같은 거로)
* 학습 과정에서 Gibbs sampling이라는 단계를 포함함.
* --> gradually adjust the weights to minimize the reconstruction error
* 무방향 노드를 가져서 양쪽 방향으로 propagation 가능함.

##
* 리뷰 논문 처음 읽어봤는데 이해하기 어렵네요
* 여러분 개강해서 힘드시죠.. 모두 화이팅입니다!!
* 저 눈물 흘리면서 논문 읽었어요...하하
* 혹시 gpt 3 논문 읽어보신 분 계신가요?? 거의 80페이지라서 엄두를 못 내고 있어요

**Comment by HYUNMIN-HWANG** [comment]
개강 힘드네요... 그래도 힘냅시다...!!! ㅠㅠ😭😭😭
저 gpt3는 아니고 gpt2.... 눈물 흘리면서 읽었던 기억이.... 그래도 이해 못하고 넘어갔던......^_ㅠ
